### DISCUSSION TITLE
Hidden children performance?

### DISCUSSION THREAD

**Discussion by Clonex** [pixijs/pixijs#8391] [Q&A] [answered]
Hello,
Do I take a performance hit if I have a lot of `AnimatedSprite` that isn't playing and is invisible, but is a child of a visible object? Or would it be smarter to remove / add them when I need them? 

Thanks,
Clonex.

**Comment by bigtimebuddy** [comment, marked as answer]
Good question.

If you are hiding the animation, you also probably want to call `stop()`. Hiding the animation will prevent it from getting rendered or the transform being called, but `stop()` will make sure the animation doesn't progress and receive frame updates (and also trigger handlers like `onLoop`, `onComplete`, etc). Once you make the animation visible again, call `play()`.

**Reply by Clonex** [reply]
> Good question.
> 
> If you are hiding the animation, you also probably want to call `stop()`. Hiding the animation will prevent it from getting rendered or the transform being called, but `stop()` will make sure the animation doesn't progress and receive frame updates (and also trigger handlers like `onLoop`, `onComplete`, etc). Once you make the animation visible again, call `play()`.

Thanks for the quick reply! 👍 
Yeah, the AnimatedSprites is stopped before switching between them. I just wanted to be sure that I didnt loose alot of performance because I always have the user animations as children on every user object.
